views/userManagementBadge.njk

```diff
@@ -15,120 +15,133 @@
-     <tr>
-        <th>Status</th>
-        <td>{{user.status}}</td>
-    </tr>
-    <tr>
-        <th>Last seen: </th>
-        <td>{{user.updatedAt}}</td>
-    </tr>
-    <tr>
-        <th>Date registered: </th>
-        <td>{{user.createdAt}}</td>
-    </tr>
-    <tr>
-        <th></th>
-        {% if user.confirmation  %}
-            <td>
-                <form id="deleteForm" action="/userManagement/deleteUser/{{user.badge}}" method="post" onsubmit="return confirm('This user will be deleted!')">
-                <button type='submit' class="btn btn-primary" name='submit' value="">Delete</button>
-                </form>
-                <br>
-            {% if user.status == "User"%}
-                <form id="ChangeStatusToManager" action="/userManagement/promoteUser/{{user.badge}}" method="post" onsubmit="return confirm('This user will be Manager!')">        
-                <button type='submit' class="btn btn-primary" name='submit' value="">Promote to Manager</button>
-                </form>
-            {% elif user.status == "Manager" %}
-                <form id="ChangeStatusToManager" action="/userManagement/demoteManager/{{user.badge}}" method="post" onsubmit="return confirm('This user will be User!')">
-                <button type='submit' class="btn btn-primary" name='submit' value="">Demote to User</button>
-                </form>
-            {% endif %}
+<div class="row">
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+        <div class="table-responsive">
+            <table  class="table table-bordered">
+                <tr>
+                    <th>Name</th>
+                    <td>{{user.first}} {{user.last}}</td>
+                </tr>
+                <tr>
+                    <th>Badge ID</th>
+                    <td>{{user.badge}}</td>
+                </tr>
+                <tr>
+                    <th>Email</th>
+                    <td>{{user.email}}</td>
+                </tr>
+                <tr>
+                    <th>Phone</th>
+                    <td>{{user.phone}}</td>
+                </tr>
+                <tr>
+                    <th>Signature</th>
+                    <td>{{user.signature}}</td>
+                </tr>
+                <tr>
+                    <th>Parent/Guardian signature</th>
+                    <td>{{user.ecSignature}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency contact name</th>
+                    <td>{{user.ecName}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency relationship</th>
+                    <td>{{user.ecRel}}</td>
+                </tr>
+                <tr>
+                    <th>Emergency phone number</th>
+                    <td>{{user.ecPhone}}</td>
+                </tr>
+                <tr>
+                    <th>Status</th>
+                    <td>{{user.status}}</td>
+                </tr>
+                <tr>
+                    <th>Updated at: </th>
```

Comment:
Looks like you missed this because this has been renamed to "Last seen:" if you look at the changes you are overriding.